lib/oli_web/live/community_live/index.ex

```diff
@@ -67,19 +73,27 @@ defmodule OliWeb.CommunityLive.Index do
     ~F"""
       <div class="d-flex p-3 justify-content-between">
         <Filter
-          change="change_filter"
-          reset="reset_filter"
-          apply="apply_filter"
-          filter={@filter}/>
+          change="change_search"
+          reset="reset_search"
+          apply="apply_search"
+          query={@query}/>
 
         <Link class="btn btn-primary" to={Routes.live_path(@socket, New)}>
           Create Community
         </Link>
       </div>
+      <div id="community-filters" class="p-3">
+        <Form for={:filter} change="apply_filter">
```

Comment:
I tried to do it that way first, but looks like the Surface `Checkbox` component only works well when it's inside a `Form` (which would make sense since it's under the Form context in Surface). For instance, the `unchecked_value` property doesn't work and produces empty data when "unclicking" the checkbox.